components/event/EventTabs.js

```diff
@@ -18,7 +18,7 @@ export function EventTabs({ tabs, eventType, onEventTypeChange }) {
           label="Select a tab"
           value={tabs.find((tab) => tab.key === eventType)?.title}
           onChange={(e) => changeTab(e)}
-          className="block w-full rounded-md border-primary-medium-low dark:bg-primary-medium dark:focus:border-secondary-low dark:focus:ring-secondary-low  focus:border-secondary-low focus:ring-secondary-low"
+          className="block w-full rounded-md border-primary-medium-low dark:bg-primary-medium dark:focus:border-secondary-low dark:focus:ring-secondary-low focus:border-secondary-low focus:ring-secondary-low px-3 py-1 mb-2"
```

Comment:
Looks good! Perhaps consider adding the classes before the `dark:` and `focus:` variants. Doesn't Prettier usually take care of that automatically?

Reply:
Thank you for the feedback, Arvind I've reordered the class names as suggested, with the dark and focus variants now at the end. As for Prettier, it doesn't seem to sort the class names automatically, so I manually rearranged them. Let me know if there's anything else I can do to improve the PR. Thanks again!

Reply:
Thanks @eugene4545!
Oh I didn't know about the extra plugin, thanks @ThomasCode92 👍🏻

Reply:
thanks @ThomasCode92 , I have applied the tailwind prettier plugin on the code to rearrange the class name

Reply:
Hey @Arvind I was wondering if there are any remaining issues that need to be addressed.

Reply:
No, I think the pr solves the mentioned issue. Let's wait for other to review it and get it merged :)